site/content/examples/21-miscellaneous/01-hacker-news/Summary.svelte

```diff
@@ -7,6 +7,14 @@
 		const c = item.comments_count;
 		return `${c} ${c === 1 ? 'comment' : 'comments'}`;
 	}
+	
+	function item_url() {
+		const { url, type } = item;
+		if (type === "ask") {
+			return `https://news.ycombinator.com/${url}`;
+		}
+		return url;
+	}
```

Comment:
This could be done more concisely with:

```
$: url = item.type === 'ask' ? `https://news.ycombinator.com/${item.url}` : item.url
```

Note that it also needs to be done in `Item.svelte` too.

Reply:
Hi @antony, thank you for your review.

As I've explained in my newer commit's message, the API seems inconsistent for 'Ask HN'-type posts. Let me know if you'd rather have both ternaries be the same.

Reply:
Ah apologies, I didn't see your later commit message. I'll defer to your knowledge of the HN API, as mine was merely from playing with it for a short while based on your PR.